test/integration/common/StudioActions.kt

```diff
@@ -236,17 +252,11 @@ object StudioActions {
             .joinToString("\n")
     }
 
-    object Errors {
-        private const val CONNECT_TYPEDB_FAILED = "Failed to connect to TypeDB."
-        private const val CREATE_DATABASE_FAILED = "Failed to create the database."
-        private const val DATA_WRITE_FAILED = "Failed to write the data."
-        private const val SCHEMA_WRITE_FAILED = "Failed to write the schema."
-    }
-
     object Delays {
         const val RECOMPOSE = 500
         const val FILE_IO = 750
         const val NETWORK_IO = 1_500
         const val CONNECT_SERVER = 2_500
+        const val WAIT_TIMEOUT: Long = 30_000
```

Comment:
It seems appropriate for us to put the 30 second timeout that `waitUntilAssertionIsTrue` uses under `Delays`. I feel that `Delays.WAIT_TIMEOUT` is clear enough, how about you?